model/model.go

```diff
@@ -99,6 +99,10 @@ type Container struct {
 	Ports        []ExposedPort `json:"ports" yaml:"ports"`
 	MemoryLimit  string        `json:"memoryLimit,omitempty" yaml:"memoryLimit,omitempty"`
 	MountSources bool          `json:"mountSources" yaml:"mountSources"`
+	// Base root command inside container
+	Command []string `json:"command" yaml:"command"`
```

Comment:
Please line up `Args` and `Command` with other fields in the struct (I also think the comments are unnecessary)